compiler/src/dmd/scanmscoff.d

```diff
@@ -53,29 +44,27 @@ void scanMSCoffObjModule(void delegate(const(char)[] name, int pickAny) pAddSymb
         eSink.error(loc, "corrupt MS-Coff object module `%s` %d", module_name, reason);
```

Comment:
`module_name` is now marked `scope` but it can be escaped by `eSink` here

Reply:
took scope off